charts/jenkins/README.md

```diff
@@ -415,6 +416,35 @@ $ helm install my-release -f values.yaml stable/jenkins
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+#### Defining Caches for your agent pods
+
+Its possible to define a set of cache volumes that can be mounted by agent pods for storing libraries and other artifacts for build. The volume defined, under `persistence`, will be created as a new `PVC` by the chart. The volume has to be mounted under the `volumes` section. Along with the cache, a `CronJob` can be defined under the `clear` section to perform cache activies, including clearing old libraries, on a periodic basis. The `CronJob` is required both to prevent unused libraries from accumulating over time and to avoid security issues.
```

Comment:
```suggestion
Its possible to define a set of cache volumes that can be mounted by agent pods for storing libraries and other artifacts for build.
The volume defined, under `persistence`, will be created as a new `PVC` by the chart.
The volume has to be mounted under the `volumes` section.
Along with the cache, a `CronJob` can be defined under the `clear` section to perform cache activities, including clearing old libraries, on a periodic basis.
The `CronJob` is required both to prevent unused libraries from accumulating over time and to avoid security issues.
```

Reply:
I have modified this and it looks cleaner. Have reworded to follow the changed naming `pvc.create` and `cronJob.create`